.tests/integration/config.yaml

```diff
@@ -80,3 +79,4 @@ svdb_merge:
 
 svdb_query:
   container: "docker://hydragenetics/svdb:2.5.0"
+  svdb_vcf: "reference/normal_26_svdb_0.8.vcf"
```

Comment:
I would rename it to just `vcf` as it is a subcategory from `svdb_query`, no need to repeat `svdb_` again 😉

Reply:
all done